packages/berry-pnpify/sources/generateSdk.ts

```diff
@@ -24,12 +24,14 @@ export async function generateSdk(projectRoot: PortablePath, targetFolder: Porta
     targetFolder = projectRoot;
 
   const tssdk = ppath.join(targetFolder, `tssdk` as PortablePath);
+  const tssdkManifest = ppath.join(tssdk, `package.json` as PortablePath);
   const tsserver = ppath.join(tssdk, `lib/tsserver.js` as PortablePath);
 
   const relPnpApiPath = ppath.relative(ppath.dirname(tsserver), ppath.join(projectRoot, `.pnp.js` as Filename));
 
   await xfs.removePromise(tssdk);
   await xfs.mkdirpPromise(ppath.dirname(tsserver));
+  await xfs.writeFilePromise(tssdkManifest, JSON.stringify({name: 'typescript', version: require('typescript/package.json').version}, null, 2));
```

Comment:
What do you think about adding a `-pnpify` prefix? This would make it easier to confirm that users are using PnPify 😃

Reply:
Yes, sounds good, will add